server/create-baseline.sql

```diff
@@ -1,35 +1,77 @@
 
-CREATE TABLE ibmdb (
-  Time varchar(100),
-  Task varchar(100),
-  b varchar(100),
-  W varchar(100),
-  G varchar(100),
-  bm varchar(100),
-  br varchar(100),
-  pn varchar(100),
-  pp varchar(100),
-  te varchar(100),
-  ie varchar(100),
-  p varchar(100),
-  TimeAndLocation varchar(10000),
-  InitialInfo varchar(10000),
-  SearchWords varchar(100),
-  finalScore varchar(100),
-  standard_info varchar(10000)
-);
-
-CREATE TABLE human_survey (
-  q1 varchar(10000),
-  q2 varchar(10000),
-  q3 varchar(10000),
-  q4 varchar(10000),
-  q5 varchar(10000),
-  q6 varchar(10000)
+CREATE TABLE game (
+  id uuid PRIMARY KEY,
+  final_score smallint,
+  start_time timestamp, -- e.g., 13:13 2018-04-22
+  total_time real -- e.g., 112.6 seconds
 );
 
-CREATE TABLE robot_survey (
+CREATE TABLE survey (
+  game_id uuid REFERENCES game(id),
   q1 varchar(10000),
```

Comment:
In `store_survey`, we only insert into `q1`. I don't think that we even need `q2` or `q3`, so should we just remove them from the table here?